docs/data/material/components/buttons/buttons.md

```diff
@@ -10,141 +10,109 @@ unstyled: /base-ui/react-button/
-
-```jsx
-<Button
-  onClick={() => {
-    alert('clicked');
-  }}
->
-  Click me
-</Button>
-```
-
-Note that the documentation [avoids](/material-ui/guides/api/#native-properties) mentioning native props (there are a lot) in the API section of the components.
+### Colors
 
-## Color
+Every palette included in the theme is available via the `color` prop.
 
 {{"demo": "ColorButtons.js"}}
 
-In addition to using the default button colors, you can add custom ones, or disable any you don't need. See the [Adding new colors](/material-ui/customization/palette/#custom-colors) examples for more info.
+:::success
+If the default colors aren't sufficient, you can add custom colors and remove defaults that you don't need.
+See the [Palette—Custom colors](/material-ui/customization/palette/#custom-colors) doc for more info.
+:::
 
-## Sizes
+### Sizes
 
-For larger or smaller buttons, use the `size` prop.
+The Button component comes in three sizes: `small`, `medium` (default), and `large`.
 
 {{"demo": "ButtonSizes.js"}}
 
-## Buttons with icons and label
+### Start and end icons
 
-Sometimes you might want to have icons for certain buttons to enhance the UX of the application as we recognize logos more easily than plain text. For example, if you have a delete button you can label it with a dustbin icon.
+Use the `startIcon` or `endIcon` prop to append an icon to the beginning or end of a Button's contents.
 
 {{"demo": "IconLabelButtons.js"}}
 
-## Icon button
+### Icon Buttons
+
+```jsx
+import IconButton from '@mui/material/IconButton';
+```
 
 Icon buttons are commonly found in app bars and toolbars.
+They're also useful for toggle buttons that give the user a single choice to be selected or deselected, such as adding or removing a star to an item.
 
-Icons are also appropriate for toggle buttons that allow a single choice to be selected or
-deselected, such as adding or removing a star to an item.
+The Material UI Icon Button component accepts the same props and values as the Button for [sizes](#sizes) and [colors](#colors).
 
 {{"demo": "IconButtons.js"}}
 
-### Sizes
-
-For larger or smaller icon buttons, use the `size` prop.
-
-{{"demo": "IconButtonSizes.js"}}
+### Handling clicks
 
-### Colors
-
-Use `color` prop to apply theme color palette to component.
-
-{{"demo": "IconButtonColors.js"}}
-
-## File upload
+Material UI Buttons accept an `onClick` handler that's applied to the root DOM element.
 
-To create a file upload button, turn the button into a label using `component="label"` and then create a visually-hidden input with type `file`.
-
-{{"demo": "InputFileUpload.js"}}
+```jsx
+<Button
+  onClick={() => {
+    alert('clicked');
+  }}
+>
+  Click me
+</Button>
+```
 
 ## Customization
 
-Here are some examples of customizing the component.
-You can learn more about this in the [overrides documentation page](/material-ui/customization/how-to-customize/).
-
-{{"demo": "CustomizedButtons.js", "defaultCodeOpen": false}}
```

Comment:
This one would be good to maintain in the Customization section, but maybe with more drastically visually-looking designs? I can help with that!

Reply:
Or not, I'm not sure, given it's just a simple visual customization without any complex/sneaky trick. What do you think? If we're not keeping it, let's remove the file!

Reply:
Awesome! I'll work on adding more later on — Material UI as a whole is lacking more entirely customized examples!